### PR TITLE
techdebt: [SMP-1590]: sync ccm charts and sync clickhouse

### PR DESCRIPTION
**Pull request by arya-harness**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/51388)

### CODE REVIEW

**Comment by arya-harness** [comment]
trigger uts

File: batch-processing/chart/values.yaml [hunk 1]
```diff
@@ -87,7 +71,7 @@ image:
   repository: harness/batch-processing-signed
   pullPolicy: Always
 # Overrides the image tag whose default is the chart appVersion.
-  tag: "78605-000"
+  tag: "80001-000"
```
Comment (iamanmolm) [inline batch-processing/chart/values.yaml:74]: We can update this to 80002-000.

File: batch-processing/chart/values.yaml [hunk 2]
```diff
@@ -97,7 +81,7 @@ imageClickhouseEnabled:
...
```
Comment (iamanmolm) [inline batch-processing/chart/values.yaml:84]: We can update this to 80002-000.

File: ce-nextgen/chart/templates/deployment.yaml [hunk 3]
```diff
@@ -40,18 +40,18 @@ spec:
         args:
           - "pod"
           - "-lapp=mongodb-replicaset"
-      {{- end }}    
+      {{- end }}
```
Comment (iamanmolm) [inline ce-nextgen/chart/templates/deployment.yaml:43]: We also need to add wait-for-clickhouse here. There is a migration that runs on start of ce-nextgen on clickhouse db.

File: ce-nextgen/chart/values.yaml [hunk 4]
```diff
@@ -76,7 +90,7 @@ image:
   repository: harness/ce-nextgen-signed
   pullPolicy: IfNotPresent
 # Overrides the image tag whose default is the chart appVersion.
-  tag: "78700-000"
+  tag: "80102-000"
```
Comment (iamanmolm) [inline ce-nextgen/chart/values.yaml:93]: We can update this to 80103-000.

File: ce-nextgen/chart/values.yaml [hunk 5]
```diff
@@ -86,7 +100,7 @@ imageClickhouseEnabled:
   repository: harness/ce-nextgen-signed
   pullPolicy: Always
 # Overrides the image tag whose default is the chart appVersion.
-  tag: "78700-000"
+  tag: "80102-000"
```
Comment (iamanmolm) [inline ce-nextgen/chart/values.yaml:103]: We can update this to 80103-000.

File: charts/bootstrap/database/README.md [hunk 6]
```diff
@@ -18,6 +19,19 @@ A Helm chart for Kubernetes
...
+| clickhouse.image.tag | string | `"23.3.4-debian-11-r0"` |  |
+| clickhouse.persistence.size | string | `"1Ti"` |  |
+| clickhouse.replicaCount | int | `1` |  |
+| clickhouse.resources.limits.memory | string | `"8Gi"` |  |
+| clickhouse.resources.requests.cpu | int | `6` |  |
+| clickhouse.resources.requests.memory | string | `"8Gi"` |  |
+| clickhouse.shards | int | `1` |  |
+| clickhouse.zookeeper.enabled | bool | `false` |  |
+| clickhouse.zookeeper.fullnameOverride | string | `"clickhouse-zookeeper"` |  |
+| clickhouse.zookeeper.replicaCount | int | `1` |  |
```
Comment (iamanmolm) [inline charts/bootstrap/database/README.md:33]: Also can we add pod label for clickhouse to be used for wait-for-container?

File: charts/bootstrap/database/values.yaml [hunk 7]
```diff
@@ -74,3 +76,157 @@ postgresql:
...
+      <prometheus>
+          <endpoint>/metrics</endpoint>
+          <port from_env="CLICKHOUSE_METRICS_PORT"></port>
+          <metrics>true</metrics>
+          <events>true</events>
+          <asynchronous_metrics>true</asynchronous_metrics>
+      </prometheus>
+      {{- end }}
+    </clickhouse>
+  replicaCount: 1
```
Comment (iamanmolm) [inline charts/bootstrap/database/values.yaml:227]: Also can we add pod label for clickhouse to be used for wait-for-container?

**Review by iamanmolm — APPROVED** [review]
Raise a different PR for the requested changes as discussed. Thanks!!!

**Comment by arya-harness** [comment]
Addressing the above comments in another PR after this is merged.

**Pull request merged**